.github/workflows/build_commits.yaml

```diff
@@ -0,0 +1,29 @@
+name: build_commits
+
+
+on: [pull_request]
+
+jobs:
+  build_commits:
+    name: build commit code
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
+      - name: install llvm and libelf
+        run: |
+          sudo apt-get install llvm
```

Comment:
I think we will need LLVM 10.0.0 specifically or we may run into issues. Maybe it's possible to reuse the cilium/cilium-builder image for that purpose?

Reply:
I am not sure about the cilium/cilium-builder image. Can I go with using "KyleMayes/install-llvm-action@v1.2.1" (already being used in bpf-checks.yaml) with version 10?

Reply:
Yeah, let's go for that since it's already used elsewhere :+1: 

Our cilium/cilium-builder image just has a couple commits on top of LLVM 10.0.0, so it should be fine.